packages/admin-ui/src/table/stories/clickable.stories.tsx

```diff
@@ -65,6 +65,7 @@ export function Clickable() {
         {
           id: 'name',
           header: 'Name',
+          width: 'minmax(100px, 2fr)',
```

Comment:
Pixel police passing through 🚓 

```suggestion
          width: 'minmax(6.25rem, 2fr)',
```

Reply:
Actually I have to remove this line, it was just a test ;) But thanks

Reply:
Oooh, ok 😉 !

Don't bother changing this on stories !